scrapers/watchingmydaughtergoblack.yml

```diff
@@ -0,0 +1,40 @@
+name: "watchingmydaughtergoblack"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - watchingmydaughtergoblack.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //h3[@class="scene-title col-xs-12"]/text()
+      Date:
+        selector: //meta[@itemprop="uploadDate"]/@content
+        postProcess:
+          - replace:
+              - regex: ([\d-]*).+
+                with: $1
+          - parseDate: 2006-01-02
+      Details:
+        selector: //p[@itemprop="description"]/text()
+      Tags:
+        Name: 
+          selector: //p[@class="categories"]/text()
+          postProcess:
+            - replace:
+              - regex: ',(\x{00A0}|\s)+'
+                with: ","
+          split: ","
+      Performers:
+        Name: //span[@class="starring-list"]/a/text()
+      Image:
+        selector: //meta[@itemprop="thumbnailUrl"]/@content
+        postProcess:
+          - replace:
+              - regex: ^
+                with: "https:"
+      Studio:
+        Name:
+          fixed: "WatchingMyDaughterGoBlack"
```

Comment:
```yaml
        selector: //div[@class="trailer-container"]//@data-img|//meta[@itemprop="thumbnailUrl"]/@content 
        postProcess:
          - replace:
              - regex: ^
                with: "https:"
      Studio:
        Name:
          selector: //meta[@itemprop="thumbnailUrl"]/@content
          postProcess:
            - replace:
                - regex: '.*/([^/]+)\.com/[^/]+$'
                  with: $1
            - map:
                barbcummings: Barb Cummings
                blackmeatwhitefeet: Black Meat White Feet
                blacksonblondes: Blacks On Blondes #ok
                blacksoncougars: Blacks On Cougars
                candymonroe: Candy Monroe
                cuckoldsessions: Cuckold Sessions #ok
                cumbang: Cum Bang #ok
                dogfartbehindthescenes: Dogfart Behind The Scenes #ok
                gloryhole-initiations: Gloryhole-Initiations
                gloryhole: Glory Hole #ok
                interracialblowbang: Interracial Blow Bang
                interracialpickups: Interracial Pickups
                katiethomas: Katie Thomas
                ruthblackwell: Ruth Blackwell
                springthomas: Spring Thomas
                theminion: The Minion
                watchingmydaughtergoblack: Watching My Daughter Go Black
                watchingmymomgoblack: Watching My Mom Go Black
                wefuckblackgirls: We Fuck Black Girls #ok
                wifewriting: Wife Writing
                zebragirls: Zebra Girls #ok
```
In some cases the thumbnail url 404s eg `https://www.watchingmydaughtergoblack.com/tour/scenes/BehindTheScenes/GloryHole_bts_lexi_kartel_maserati_xxx/`, using the player image as the first selector seems to work ok 
For the studio i verified the ones with ok, i will update when i get the chance to test the rest